src/Compilers/CSharp/Portable/Compilation/CSharpSemanticModel.cs

```diff
@@ -1852,7 +1852,7 @@ private bool IsInTypeofExpression(int position)
         private static void AddUnwrappingErrorTypes(ArrayBuilder<Symbol> builder, Symbol s)
         {
             var originalErrorSymbol = s.OriginalDefinition as ErrorTypeSymbol;
-            if ((object)originalErrorSymbol != null)
+            if ((object)originalErrorSymbol != null && originalErrorSymbol.CandidateSymbols != null)
```

Comment:
If `ErrorTypeSymbol.CandidateSymbols` is returning `default(IImmutableArray<Symbol>)`, it seems like the bug is in `ErrorTypeSymbol`.